### PR TITLE
chore: Publish JS package to npm

### LINKED ISSUE #35: Publish JavaScript proto generated files (controlclient.proto/cacheclient.proto) to npm
Currently, we're publishing generated gPRC code to Artifactory. 
Since we're publishing the JS SDK to npm, we also want JS generated gRPC code to npm.

cc @danielamiao

### PR DESCRIPTION
**Pull request by poppoerika**
Manually published this package to npm: https://www.npmjs.com/package/@momentohq/generated-types
This pr is to update the javascript-publish step so that it'll publish to npm instead of Artifactory.

Closes #35

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -60,16 +56,16 @@ jobs:
           popd
         shell: bash
 
-      - name: Publish npm package to Artifactory
+      - name: Publish npm package
         run: |
-          export VERSION="${{needs.release.outputs.version}}"
+          export VERSION="${{ needs.release.outputs.version }}"
           echo "Publishing package with version $VERSION"
-          pushd javascript
```
Comment (kvcache) [inline .github/workflows/push.yml:67]: Doesn't this deletion break the workflow?

Comment (poppoerika) [reply .github/workflows/push.yml:67]: Good call. Thank you for catching this!

**Comment by poppoerika** [comment]
Newly published package: https://www.npmjs.com/package/@momentohq/generated-types
The old package is removed from npm.

**Review by kvcache — APPROVED** [review]
Lgtm

**Pull request merged**